app/views/tag-manager/_body-content.njk

```diff
@@ -0,0 +1,7 @@
+{% if googleTagManagerKey %}
```

Comment:
I think the content in the body partial should be in the head and vice versa, at least based on the instructions [here](https://developers.google.com/tag-platform/devguides/add-tag?hl=en&solution=gtm).